adapters/taboola/taboola.go

```diff
@@ -238,17 +243,17 @@ func createTaboolaRequests(request *openrtb2.BidRequest) (taboolaRequests []*ope
 	return taboolaRequests, errs
 }
 
-func makeRequestExt(pageType string, errs []error) json.RawMessage {
+func makeRequestExt(pageType string) (json.RawMessage, error) {
 	requestExt := &RequestExt{
 		PageType: pageType,
 	}
 
 	requestExtJson, err := json.Marshal(requestExt)
 	if err != nil {
-		errs = append(errs, err)
-		return make([]byte, 0)
+		fmt.Errorf("could not marshal %s", requestExt)
+		return nil, err
```

Comment:
Optional suggestion - this is correct but it could be simply be one liner - 
```
return nil, fmt.Errorf("could not marshal %s", requestExt)
```

Reply:
Thanks, please approve